### PR TITLE
5122 Add newsletter signup form

### LINKED ISSUE #5122: /blog integration: add newsletter sign-up
## Summary

Add the newsletter signup card to the `/blog/index.html` template

## Screenshot

image

### PR DESCRIPTION
**Pull request by solazio**
## Done

Add newsletter signup form

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/blog](http://0.0.0.0:8001/blog)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure the form works and it behaves/looks the same as the one on https://blog.ubuntu.com/

## Issue / Card

Fixes #5122

### CODE REVIEW

**Comment by solazio** [comment]
@pmahnke could you please confirm if the marketo part of this form is the same as the one used on https://blog.ubuntu.com/

**Comment by anasereijo** [comment]
- [ ]  I don't get a thank you page

- [ ] capitalise 'Server'

**Comment by deadlight** [comment]
If I trigger a validation error then the input becomes smaller:
image

It stays small when I start entering an address:
image

But then suddenly gets larger after a few characters have been entered:
image


I think that it should be the larger size in all cases.

(FF on Ubuntu)

**Comment by anasereijo** [comment]
@solazio 👍

File: templates/blog/newsletter-form.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        <div class="mktField mktLblRight u-margin-medium--top">
+          <span class="mktInput mktLblRight">
+            <input class="u-no-margin--top mktFormCheckbox" name="canonicalUpdatesOptIn" id="CanonicalUpdatesOptIn" value="Receive updates" type="checkbox" />
+            <label for="CanonicalUpdatesOptIn" class="p-form__label u-no-margin--top">
+              <small>I agree to receive information about Canonical’s products and services.</small>
+            </label>
+            <span class="mktFormMsg"></span>
+          </span>
+        </div>
+          <div class="g-recaptcha" data-sitekey="6LfYBloUAAAAAINm0KzbEv6TP0boLsTEzpdrB8if" style="margin: 2rem 0;"></div>
```
Comment (deadlight) [inline templates/blog/newsletter-form.html]: Fix indent

**Pull request merged**